src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractRabbitBatchSubscriber.java

```diff
@@ -78,15 +77,35 @@ protected MB filter(MB batch) {
 
     protected abstract Metadata extractMetadata(M message);
 
-
-    @Getter
-    @Builder
     protected static class Metadata {
         private long sequence;
         private String messageType;
         private String sessionAlias;
         private Direction direction;
 
+        public Metadata(long sequence, String messageType, String sessionAlias, Direction direction) {
```

Comment:
Maybe we should reorder the `messageType` and `sessionAlias` parameters in the constructor to avoid two String parameters next to each other.
```suggestion
        public Metadata(long sequence, String messageType, Direction direction, String sessionAlias) {
```